sdk/core/Microsoft.ClientModel.TestFramework/tsp/main.tsp

```diff
@@ -0,0 +1,255 @@
+import "@typespec/http";
```

Comment:
https://github.com/Azure/azure-sdk-tools/blob/main/tools/test-proxy/typespec/main.tsp

Reply:
Should we use this?

Reply:
Ooh I didn't know that was there. I'll try generating from it to see if it works

Reply:
Nor did I until I saw this issue - https://github.com/Azure/azure-sdk-for-net/issues/51797